docs/extensions-framework/getting-started.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Getting started"
+date: 2020-05-28T10:39:00+01:00
+weight: 1
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs
+geekdocFilePath: extensions-framework/getting-started.md
+---
+
+{{< toc >}}
+
+## Create own extension
+- [Create frontend]({{< ref "create-frontend.md" >}})
```

Comment:
One sentence before the link would be cool: 
`When creating your own frontend extension, ocis-web has some requirements and options for the extension integration. Learn more about` and then the link with name `Creating a frontend extension`